app/assets/stylesheets/_pages/_commits.scss

```diff
@@ -287,7 +287,9 @@
   border-color: #333;
 
   .status__icon {
```

Comment:
Maybe I'm misunderstanding how CSS rules work, but won't this also change the icon for the actual error status?

Reply:
Yeah you're right. We should split the rule for error & timedout, nice catch.

Reply:
Fixed.